docs/INSTALL.md

```diff
@@ -49,7 +49,8 @@ tool installations are described in [Advanced](#Advanced).
 **Installing F´ Python Packages**
 
 ```
-pip install fprime/
+pip install -U setuptools wheel pip
+pip install -U -r fprime/requirements.txt
```

Comment:
Any particular reason to use the `-U` flag here? If a python dependency makes a breaking change, I feel like that kind of defeats the point of using a `requirements.txt` for reproducible builds.